package.json

```diff
@@ -227,12 +219,9 @@
     "opn": "^5.5.0",
     "oppsy": "^2.0.0",
     "pegjs": "0.10.0",
-    "postcss-loader": "^3.0.0",
     "prop-types": "15.6.0",
     "proxy-from-env": "1.0.0",
-    "pug": "^2.0.4",
```

Comment:
FYI there is an unused pug file in `src/legacy/server/views` that can be deleted. I believe this PR can now close this issue as well: https://github.com/elastic/kibana/issues/53160